go-controller/pkg/ovn/egressqos.go

```diff
@@ -439,11 +458,6 @@ func (oc *DefaultNetworkController) syncEgressQoS(key string) error {
 		klog.V(4).Infof("Finished syncing EgressQoS %s on namespace %s : %v", name, namespace, time.Since(startTime))
 	}()
 
-	eq, err := oc.egressQoSLister.EgressQoSes(namespace).Get(name)
-	if err != nil && !apierrors.IsNotFound(err) {
-		return err
```

Comment:
why is this logic going away? if the object is not found you still want to sync?

Reply:
yes, we just moved this logic one level up in  `processNextEgressQoSWorkItem`, this would still remove QoS related objects from OVN for delete case.